wordpress-mobile/WordPress-iOS
# Google Signup: Replacing image shown while waiting for Google.

**Ref:** #8109 

**To test:**
- Log out if necessary.
- Select 'Signup to WordPress.com'.
- Select 'Sign up with Google'.
- On the two screens in this flow waiting for Google to do stuff, the image should be:
<img width="241" alt="google_waiting" src="https://user-images.githubusercontent.com/1816888/36622839-0604efbe-18bd-11e8-89a5-bc646d34077f.png">
